Duplicati/CommandLine/Commands.cs

```diff
@@ -133,11 +133,7 @@ public static int Examples(TextWriter outwriter, Action<Duplicati.Library.Main.C
     
         public static int Help(TextWriter outwriter, Action<Duplicati.Library.Main.Controller> setup, List<string> args, Dictionary<string, string> options, Library.Utility.IFilter filter)
         {
-            if (args.Count < 1)
-                Duplicati.CommandLine.Help.PrintUsage(outwriter, "help", options);
-            else
-                Duplicati.CommandLine.Help.PrintUsage(outwriter, args[0], options);
-                
+            Duplicati.CommandLine.Help.PrintUsage(outwriter, args.Count == 0? "help" : args[args.Count - 1], options);

```

Comment:
This changes the logic a bit, as it would previously use the first argurment, but now it uses the last argument.
I don't have a preference for either, but was wondering if there was an idea behind the change?

Reply:
Thanks for pointing this out. I did realize this but at the time didn't come up with a better solution. I added a commit that takes the first argument after help if present, based on the knowledge that the first argument, if any, is always help (otherwise we wouldn't have arrived in this function).